lib/controller.js

```diff
@@ -170,6 +177,134 @@ Tessel.get = function(opts) {
     }
   });
 };
+/*
+Because of sync problem with master trunk and parallel development on Tessel.list 
+it was necessary to go my own way...
```

Comment:
Ideally, we should remove the entire `seek` method because this functionality is in master.